alf/examples/off_policy_actor_critic.py

```diff
@@ -0,0 +1,81 @@
+# Copyright (c) 2019 Horizon Robotics. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Off-policy training using ActorCriticAlgorithm.
```

Comment:
Maybe it would be better to limit 'offpolicyness' from too high if we want implement a generic off-policy actor critic algorithm. Like simply remove old experiences in replay buffer after some time-steps.

Reply:
That can be done in separate change.